ui-common/src/components/TrafficViewer/EntriesList.tsx

```diff
@@ -109,14 +116,26 @@ export const EntriesList: React.FC<EntriesListProps> = ({
     if (scrollTo) {
       scrollableRef.current.scrollToIndex(data.data.length - 1);
     }
-  }, [setLoadMoreTop, setIsLoadingTop, entries, setEntries, query, setNoMoreDataTop, leftOffTop, setLeftOffTop, setQueriedTotal, setTruncatedTimestamp, scrollableRef]);
+
+    return oldEntries
+  }, [trafficViewerApi, setLoadMoreTop, setIsLoadingTop, entries, setEntries, query, setNoMoreDataTop, leftOffTop, setLeftOffTop, setQueriedTotal, setTruncatedTimestamp, scrollableRef]);
 
   useEffect(() => {
     if (!isWsConnectionClosed || !loadMoreTop || noMoreDataTop) return;
     getOldEntries();
   }, [loadMoreTop, noMoreDataTop, getOldEntries, isWsConnectionClosed]);
 
-  const scrollbarVisible = scrollableRef.current?.childWrapperRef.current.clientHeight > scrollableRef.current?.wrapperRef.current.clientHeight;
+  useEffect(() => {
+    (async () => {
+      if (isShouldStartStreamData && trafficViewerApi?.fetchEntries) {
+        setEntries([])
+        const oldEntries = await getOldEntries()
+        const leffOffButton = oldEntries.length > 0 ? oldEntries[oldEntries.length - 1].id : DEFAULT_LEFTOFF
```

Comment:
leffOffBottom?

Reply:
and why do you need to recalculate here the leftOffBottom if you already have it at the top of the component?